routemaster/cli.py

```diff
@@ -19,10 +23,38 @@
     type=click.File(encoding='utf-8'),
     required=True,
 )
+@click.option(
+    '-l',
+    '--log-level',
+    help="Logging level.",
+    type=click.Choice((
+        'CRITICAL',
+        'ERROR',
+        'WARNING',
+        'INFO',
+        'DEBUG',
+    )),
+    default='INFO',
+)
 @click.pass_context
-def main(ctx, config_file):
+def main(ctx, config_file, log_level):
     """Shared entrypoint configuration."""
-    config = load_config(yaml.load(config_file))
+    logging.basicConfig(
+        format=(
+            "[%(asctime)s] [%(process)d] [%(levelname)s] "
+            "[%(name)s] %(message)s"
+        ),
+        datefmt="%Y-%m-%d %H:%M:%S %z",
+        level=getattr(logging, log_level),
+    )
+
+    try:
+        config = load_config(yaml.load(config_file))
+    except ConfigError as e:
+        msg = f"Configuration Error: {e}"
+        logger.exception(msg)
```

Comment:
I'm not entirely convinced that these start-up errors should go (only) to the log. While we're outputting only to stdout it's probably fine, but if we start doing other things with the logging we'll probably want to review that.

Reply:
I'm going to leave this one for now, I'm not as familiar with logging as I'd like and not certain what the best option is, plus I'm hoping to refactor logging at some point anyway.